docker-compose.yml

```diff
@@ -97,4 +102,9 @@ secrets:
   aws_s3_secret:
     file: aws_s3_secret.sec
   aws_s3_access:
-    file: aws_s3_access.sec
\ No newline at end of file
+    file: aws_s3_access.sec
+configs:
+  ingest_config:
+    file: ./configs/ingest_config.yml
+  mis_config:
+    file: ./configs/mis_config.yml
```

Comment:
❓Seems weird that we need to repeat the path to each config file three times

Reply:
The paths under the `configs` section are local and the the `target` paths refer to the containers.